lib/config/.rubocop.yml

```diff
@@ -1,3 +1,8 @@
+require:
```

Comment:
Propose to move this file to `config/rubocop` folder and rename him to `config.yml`